CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Welcome contributors to Alva
+💓 First of all, we'd love for you to contribute to our source code. 💓
+
+## Code of Conduct
+We belive in a welcoming and inclusive environment / community, that's why we enforce our Code of Conduct.
+For more information -> [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md).
+
+
+## Table of Contents
+- [Link collection](#short-collection-of-links-to-important-resources)
+- [Setup for contributers](#setup-for-contributers)
+- [Issues](#issues)
+	- [Good first issue for Contributors](#good-first-issue)
+	- [Lable structur](#labels)
+	- [Working on issues](#working-on-issues)
+- [Style Guide / Coding conventions](#style-guide-/-coding-conventions)
+	- [Coding conventions & guidelines](#coding-conventions-&-guidelines)
+
+
+## Short collection of Links to Important Resources
+   * bugs / feature requests: https://github.com/meetalva/alva/issues
+   * website: https://meetalva.io
+   * sprint board: https://github.com/meetalva/alva/projects/2
+   * backlog board: https://github.com/meetalva/alva/projects/3
+
+
+## Setup for contributers
+To add a contribution to Alva (improve its features, fix issues),
+get the source of both the application itself,
+and the kickstart styleguide (designkit):
+
+* Clone this repository [alva](https://github.com/meetalva/alva): git@github.com:meetalva/alva.git
+* Clone [designkit](https://github.com/meetalva/designkit) repository: git@github.com:meetalva/designkit.git
```

Comment:
I think i should bring more context to this.
The designkit should not really be needed for the development. It is just a example project.

If the contributer already has a lsg that is compatible with alva there is no need to use the designkit.
If you want i can try to rewrite this part